### PR TITLE
[PG-1428] No decline option for posting policy

### PR DESCRIPTION
**Pull request by avi-bh**
### JIRA Ticket
https://bighealth.atlassian.net/browse/PG-1428

### Problem
* Remove "decline" option from Posting Policy (ToS)

### Solution
* Comment out button for declining ToS. Opted for commenting out instead of deleting the code to clarify our changes to Misago's original code. Happy to delete if that's the consensus.

#### Manual Tests
* Manually tested in Dev.

### CODE REVIEW

**Review by wsbastian — DISMISSED** [review]
LGTM. 

Do we have any deps on the PG-1376 branch for this, or can we set this PR up against master?

**Comment by avi-bh** [comment]
@wsbastian I've been branching off the PG-1376 branch to avoid conflicts later. I'll change this to target master after that is merged (or else the diff is too noisy).

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Change Log
+All notable changes to this project will be documented in this file.
+
+## 2021-02-06
+
+### Changed
+- [PG-1428](https://bighealth.atlassian.net/browse/PG-1428) Remove 'decline' button from posting policy ([#13](https://github.com/sleepio/community-app-web/pull/13))
```
Comment (jjosephbh) [inline CHANGELOG.md:7]: Nice

**Review by jjosephbh — APPROVED** [review]
LGTM

**Review by wsbastian — APPROVED** [review]

**Pull request merged**